learn/best_practices/multidim_arrays.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: book
+title: Multidimensional Arrays
+permalink: /learn/best_practices/multidim_arrays
+---
+
+Always access slices as `V(:, 1)`, `V(:, 2)`, or `V(:, :, 1)`, e.g. the
```

Comment:
Suggestion: addition of a section that explain how arrays are stored in Fortran. So the user will beter understand why it would be fast/slow on how it is accessed